src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -19,4 +24,43 @@ public boolean connect(String database, String user, String password) {
             return false;
         }
     }
+
+    public String[] getTableNames() {
+        String[] tableNames = new String[0];
+        try {
+            if (connection != null && connection.isValid(VALID_TIMEOUT)) {
+                try (Statement statement = connection.createStatement();
+                     ResultSet resultSet = statement.executeQuery(TABLE_NAMES_QUERY)) {
+                    if (resultSet.isBeforeFirst()) {
+                        int arraySize = getTableRowsCount("pg_stat_user_tables");
+                        tableNames = new String[arraySize];
+                        for (int index = 0; resultSet.next(); index++) {
+                            tableNames[index] = resultSet.getString(1);
+                        }
+                    }
+                }
+            } else {
+                throw new SQLException("The connection to the database is not established");
+            }
+        } catch (SQLException e) {
+            e.getMessage();
+        }
+        return tableNames;
+    }
+
+    private int getTableRowsCount(String tableName) throws SQLException {
+        try (Statement statement = connection.createStatement();
+             ResultSet resultSet = statement.executeQuery(TABLE_ROWS_COUNT_QUERY.replace("?",tableName))) {
+            resultSet.next();
+            return resultSet.getInt(1);
+        }
+    }
+
+    public void closeConnection() {
+        try {
+            connection.close();
+        } catch (SQLException e) {
+            e.getMessage();
```

Comment:
This handling will give you the same result as empty one. I believe that you've forgot sout() method, but anyway it's better not to handle this exception here, it should go upper, probably as a new exception

Reply:
I really forgot sout() :)